### PR TITLE
Added Human Made as a contributor

### PR DESCRIPTION
**Pull request by dashaluna**
Issue #87

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [![Build Status](https://travis-ci.org/dashaluna/hm-content-toc.svg?branch=master)](https://travis-ci.org/dashaluna/hm-content-toc)
 
 # HM Content TOC #
-**Contributors:** dashaluna, tcrsavage, johnbillion, sanchothefat  
+**Contributors:** dashaluna, tcrsavage, johnbillion, sanchothefat, humanmade  
 **Tags:** TOC, TOC shortcode, shortcode, content TOC, post TOC, page TOC, TOC for content, table of content, table of contents, HM, Human Made  
 **Requires at least:** 4.2  
 **Tested up to:** 4.3  
-**Stable tag:** 1.0.1
+**Stable tag:** 1.0.1
```
Comment (dashaluna) [inline README.md:8]: @johnbillion Do I change the version before the release of that version or when I start work on something new? Does it make sense?

**Comment by dashaluna** [comment]
@johnbillion review & merge

**Comment by johnbillion** [comment]
So the main issue here is that it doesn't make sense to release a new version of the plugin (1.0.2) just because there were some changes made to the readme. It looks like the deployment script doesn't support deploying an update to an existing version (which makes sense, you shouldn't really update an existing release even if it's just to update the readme file).

You can either leave these changes until you do some code updates too, and then release 1.0.2, or you can use `svn` on the command line to update the readme file manually.

**Comment by dashaluna** [comment]
@johnbillion alright, I see. I meant in general. When do I change the version number - just before the release or when I start working on the new feature?

I wanted to rename the plugin as per #86 and then release all these tweaks together.

**Comment by johnbillion** [comment]
Yeah changing the version number should be the final thing you do before releasing the update. You can change the version number to `1.0.2-beta` in the meantime, if you like, but it's not really necessary.

**Comment by johnbillion** [comment]
I'll merge this because it doesn't actually change the version number, just adds a readme entry for the new version.

**Pull request merged**